src/Cryptol/TypeCheck/Solver/Numeric/Sampling/Q'.hs

```diff
@@ -0,0 +1,47 @@
+{-# LANGUAGE NamedFieldPuns #-}
+{-# LANGUAGE RecordWildCards #-}
+
+module Cryptol.TypeCheck.Solver.Numeric.Sampling.Q' where
```

Comment:
What is this numeric type? Why do we need it?

Reply:
turns out not needed